common/zoom/zoom-authorization.ts

```diff
@@ -12,9 +13,17 @@ const apiSecret = process.env.ZOOM_API_SECRET;
 const grantType = 'account_credentials';
 const accountId = process.env.ZOOM_ACCOUNT_ID;
 
+if (isZoomFeatureActive()) {
+    assert(apiKey, 'Missing ZOOM_API_KEY in ENV');
+    assert(apiSecret, 'Missing ZOOM_API_SECRET in ENV');
+    assert(accountId, 'Missing ZOOM_ACCOUNT_ID in ENV');
+}
+
 let accessToken: string | null = null;
 
 const getAccessToken = async (scope?: string) => {
+    assureZoomFeatureActive();
```

Comment:
All API accesses are guarded, to ensure the feature toggle is properly used everywhere